mps-model-adapters-plugin/src/test/kotlin/org/modelix/model/mpsadapters/CreateProjectFromScratch.kt

```diff
@@ -0,0 +1,62 @@
+package org.modelix.model.mpsadapters
+
+import org.modelix.model.api.BuiltinLanguages
+
+/*
+ * Copyright (c) 2023.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+class CreateProjectFromScratch : MpsAdaptersTestBase() {
```

Comment:
Can we follow typical test class naming conventions here? Would make identifying test classes easier.

Reply:
Renamed to `CreateProjectFromScratchTest`.